noun/retrieve.c

```diff
@@ -1352,6 +1352,21 @@ u3r_words(c3_w    a_w,
   }
 }
 
+/* u3r_chubs():
+**
+**  Copy double-words (a_w) through (a_w + b_w - 1) from (d) to (c).
+*/
+void
+u3r_chubs(c3_w    a_w,
+          c3_w    b_w,
+          c3_d*   c_d,
+          u3_atom d)
+{
+  /* XX: assumes little-endian
```

Comment:
I think our whole system assumes little-endian, so this should be fine.

Reply:
although maybe @eglaysher feels differently ...

Reply:
This is an exact copy from cc-release: https://github.com/urbit/urbit/blob/cc-release/noun/retrieve.c#L1593. I guess I could've found the relevant commit and cherry-picked it.